cmd/mothership/mothership/start/http.go

```diff
@@ -110,8 +113,16 @@ func startWebserver(ctx context.Context, o *Options) error {
 		fmt.Sprintf("/v{%s}/clusters/{%s}/config/{%s}", paramContractVersion, paramRuntimeID, paramConfigVersion),
 		callHandler(o, getKymaConfig)).Methods(http.MethodGet)
 
+	apiRouter.HandleFunc(
```

Comment:
I thought we agreed to use a interval based occupancy updating on the mothership side - why do we need these endpoints?